BP/localization/BP_titles_cultural_names_l_english.yml

```diff
@@ -0,0 +1,480 @@
+﻿l_english:
+ ##### Religious Titles
+ ##### Roman Empire
+ #### e_roman_empire
+ cn_imperium_romanum:0 "Imperium Romanum"
+ cn_basileia_rhomaion:0 "Basileía Rhōmaíōn"
+ #### e_wre
+ cn_imperium_romanum_occidentale:0 "Imperium Romanum Occidentale"
```

Comment:
I'm not entirely sure it is a good idea to go full Latin on the names, I think the WRE shouldn't be renamed. However, if this is to be used, it should not be _Imperium Romanum Occidentale_ to my knowledge but rather _Pars Occidentalis_. I don't read Latin, but historically it was _pars_ that was used.

Reply:
> However, if this is to be used, it should not be Imperium Romanum Occidentale to my knowledge but rather Pars Occidentalis. I don't read Latin, but historically it was pars that was used.

That was because there weren't two empires but one empire with two courts. Pars Occidentlis is the Western Part/Portion [of the Empire].

Reply:
> > However, if this is to be used, it should not be Imperium Romanum Occidentale to my knowledge but rather Pars Occidentalis. I don't read Latin, but historically it was pars that was used.
> 
> That was because there weren't two empires but one empire with two courts. Pars Occidentlis is the Western Part/Portion [of the Empire].

It doesn't make sense for the cultural title localisation to create a distinction which didn't exist legally at the time. Anyway, as I said I don't think this debate is justified, it is better to keep the English WRE.

Reply:
Removed WRE localization for the time being while newer localization is settled upon. We should (eventually) see if custom localization or localization triggers can be used to rename titles; that'd probably produce the results we want (if possible).